config/index.js

```diff
@@ -6,7 +6,7 @@ import defaultConfig from './default.json'
 
 const debug = makeDebug('config')
 
-let env = process.env.NODE_ENV ? `${process.env.NODE_ENV}.config` : 'dev.config'
+let env = process.env.NODE_ENV
```

Comment:
I think I'd like to clean up this whole config file in another PR and just use the `config` package as is. There seems like there's a lot of code here that can easily become bugs just like this.

Also the `debug` setup is due for a refactoring. The bug here wasn't being surfaced since it was being swallowed up in the `debug`.

Reply:
Yeah, this file is a bit messy. Only comment would be that there's a mention of `dev.config.json` on line 21 which is not valid any more, but doesn't matter if we clean up this file soon anyway.